tests/unit/conditions/test_jwt_condition.py

```diff
@@ -0,0 +1,146 @@
+from datetime import datetime, timezone
+
+import jwt
+import pytest
+from marshmallow import validates
+
+from nucypher.policy.conditions.base import ExecutionCall
+from nucypher.policy.conditions.jwt import JWTCondition, JWTVerificationCall
+
+TEST_ECDSA_PRIVATE_KEY_RAW_B64 = (
+    "MHcCAQEEIHAhM7P6HG3LgkDvgvfDeaMA6uELj+jEKWsSeOpS/SfYoAoGCCqGSM49\n"
+    "AwEHoUQDQgAEXHVxB7s5SR7I9cWwry/JkECIRekaCwG3uOLCYbw5gVzn4dRmwMyY\n"
+    "UJFcQWuFSfECRK+uQOOXD0YSEucBq0p5tA=="
+)
+
+TEST_ECDSA_PRIVATE_KEY = (  # TODO: Workaround to bypass pre-commit hook that detects private keys in code
+    "-----BEGIN EC"
+    + " PRIVATE KEY"
+    + f"-----\n{TEST_ECDSA_PRIVATE_KEY_RAW_B64}\n-----END EC"
+    + " PRIVATE KEY-----"
+)
+
+TEST_ECDSA_PUBLIC_KEY = (
+    "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEXHVxB7s5SR7I9cWwry"
+    "/JkECIReka\nCwG3uOLCYbw5gVzn4dRmwMyYUJFcQWuFSfECRK+uQOOXD0YSEucBq0p5tA==\n-----END PUBLIC "
+    "KEY-----"
+)
+
+ISSUED_AT = datetime.now(tz=timezone.utc)
+
+TEST_JWT_TOKEN = jwt.encode(
+    {"iat": ISSUED_AT}, TEST_ECDSA_PRIVATE_KEY, algorithm="ES256"
+)
+
+
+def jwt_token(with_iat: bool = True, claims: dict = None):
+    claims = claims or dict()
+    if with_iat:
+        claims["iat"] = ISSUED_AT
+
+    return jwt.encode(claims, TEST_ECDSA_PRIVATE_KEY, algorithm="ES256")
+
+
+class TestJWTVerificationCall(JWTVerificationCall):
```

Comment:
How come we need this test call class? Is that because you are directly testing the call as opposed to having the condition execute the call?

Reply:
Yes, I guess a different way could have been to mock the validation method. At that time, I just found this more natural.

Reply:
Defintely not an issue; just curious. 

Basically, the condition executes the underlying call, and then compares the result with a return value test. So calling `verify()` on the condition implicitly tests `execute()` on the call. So, you may end up with repeated tests if you test the call and condition separately.